modules/dxp/apps/commerce-demo-pack/commerce-theme-minium/commerce-theme-minium-full-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/full/site/initializer/internal/dependencies/theme-portlet-settings.json

```
@@ -36,7 +36,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
+				"FileName": "minium_sub_site_navigation.ftl",
```

Don't think this was meant to be changed.

```suggestion
				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
```